src/boat_simulator/boat_simulator/nodes/physics_engine/physics_engine_node.py

```diff
@@ -120,6 +121,12 @@ def __init_private_attributes(self):
             generator=MVGaussianGenerator(np.array([1, 1]), np.array([[2, 1], [1, 2]]))
         )
 
+    def __update_wind_sensor(self):
+        """Updates the wind sensor with the latest wind data from the wind generator."""
+        wind_data = self.__wind_generator.next()
+        self.__wind_sensor.speed.speed = wind_data[0]
+        self.__wind_sensor.direction = wind_data[1]
```

Comment:
`direction` doesn't look like a field in the `WindSensor` class, you'll have to add it by following the pattern of other fields in these classes (ex: `wind` field)